lambdas/src/utils/SmartContentClient.ts

```diff
@@ -142,7 +141,13 @@ export class SmartContentClient implements ContentAPI {
       } catch {
         SmartContentClient.LOGGER.info('Defaulting to external content server url: ', contentClientUrl)
       }
-      this.contentClient.resolve(new ContentClient(contentClientUrl, 'lambdas'))
+      this.contentClient.resolve(
+        new ContentClient({
+          contentUrl: contentClientUrl,
+          origin: 'lambdas',
+          proofOfWorkEnabled: this.proofOfWorkEnabled
```

Comment:
If the content url is the internal one, then we shouldn't enable PoW. Since we are not going through nginx at all
```suggestion
          proofOfWorkEnabled: contentClientUrl === this.externalContentServerUrl && this.proofOfWorkEnabled
```